fire_atlas_editor/lib/screens/editor_screen/widgets/selection_canvas/canvas_sprite.dart

```diff
@@ -81,10 +81,8 @@ class _CanvasSpritePainer extends CustomPainter {
   @override
   void paint(Canvas canvas, Size size) {
     // Background
-    canvas.drawRect(
-        Rect.fromLTWH(0, 0, size.width, size.height),
-        Paint()
-          ..color = TinyColor(_gridTileColor.withOpacity(1)).lighten(60).color);
+    canvas.drawRect(Rect.fromLTWH(0, 0, size.width, size.height),
+        Paint()..color = _gridTileColor.withOpacity(1).brighten(0.6));
```

Comment:
missing trailing comma

Reply:
Oh, the famous trailing comma :P

Changed